data/json/npcs/talk_tags.json

```diff
@@ -68,6 +68,75 @@
     "category": "<name_b>",
     "//": "generic negative/hostile pronouns",
     "text": [
+      "airhead",
+      "arse",
+      "ass",
+      "assclown",
+      "assface",
+      "asstown",
+      "asswagon",
+      "bitch",
+      "bootlicker",
+      "bugger",
+      "butchery refuse",
+      "butthead",
+      "butt-licker",
+      "chucklefuck",
+      "coward",
+      "cuck",
+      "cunt",
+      "damn <name_b>",
+      "damned <name_b>",
+      "dildo",
+      "dimwit",
+      "dog",
+      "dumbshit",
+      "dumpster",
+      "dumpster fire",
+      "dunderfuck",
+      "effing <name_b>",
+      "emergency ration",
+      "empty headed <name_b>",
+      "expendable <name_b>",
+      "fart",
+      "fat-ass",
+      "frail-ass <name_b>",
+      "frigger",
+      "fungus eater",
```

Comment:
I think you should not add any content-specific swears- they don't feel right in full conversion mods (which won't have mentioned monsters).